src/integrationtests/java/com/aws/greengrass/integrationtests/mqttclient/MqttClientPublishTest.java

```diff
@@ -101,6 +109,22 @@ void before() throws IOException, InterruptedException, ExecutionException {
         lenient().when(connection.disconnect()).thenReturn(CompletableFuture.completedFuture(null));
         lenient().when(connection.publish(any(), any(), anyBoolean()))
                 .thenReturn(CompletableFuture.completedFuture(0));
+        AwsIotMqtt5ClientBuilder mockMqtt5Builder = mock(AwsIotMqtt5ClientBuilder.class, Answers.RETURNS_SELF);
```

Comment:
This works because MQTT5 client is temporarily used by default? Won't we want separate/new tests for it eventually?

Reply:
This works because the mocks for the mqtt 3 client are present too. But yes, these will need to be made into parameterized tests to exercise both clients.